core/crypto/bls_provider/bls_types.hpp

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CRYPTO_BLS_PROVIDER_TYPES_HPP
+#define CRYPTO_BLS_PROVIDER_TYPES_HPP
+
+#include <array>
+
+#include "common/outcome.hpp"
+
+namespace fc::crypto::bls {
+
+  using PrivateKey = std::array<uint8_t, 32>;
```

Comment:
are there constants with private key, public key and signature sizes from bls library? If so can we use them instead of magic numbers?

Reply:
There are some:
https://github.com/filecoin-project/cpp-filecoin/pull/15/files#diff-16aff5edd15d86e03435647aa6164dabR8-R14. Use them instead of magic numbers